ipn/ipnlocal/local.go

```diff
@@ -4508,14 +4516,34 @@ func (b *LocalBackend) ShouldRunSSH() bool { return b.sshAtomicBool.Load() && en
 // call regardless of whether b.mu is held or not.
 func (b *LocalBackend) ShouldRunWebClient() bool { return b.webClientAtomicBool.Load() }
 
-func (b *LocalBackend) setWebClientAtomicBoolLocked(nm *netmap.NetworkMap, prefs ipn.PrefsView) {
-	shouldRun := prefs.Valid() && prefs.RunWebClient()
+// ShouldExposeWebClient reports whether the web client should accept
+// connections via [tailscale IP]:5252 in addition to the default
+// behaviour of accepting local connections over 100.100.100.100.
+func (b *LocalBackend) ShouldExposeWebClient() bool {
```

Comment:
ShouldExposeRemoteWebClient maybe? I guess "expose" is the operative word here but that distinction wasn't obvious until I read the docs.

Reply:
Ahh good call, adding `Remote` makes things more obvious. Will add this in.